### ISSUE TITLE
a11y issues on product detail page

### ISSUE THREAD

**Issue by Stofftiger** [SAP/spartacus#12415]
product example with wrong behavior: Digital Camera Professional Black and  Digital Camera Professional Metallic 

working example: Camileo S10 EU

missing red focus frame for the second picture.
Red frame for the first and third picture is permanent, while the the frame for the second picture stays only while the curser hovers the preview picture. Also the third preview picture is never shown as big picture. If the third preview picture is selected, the big view of the second picture is shown. 
probably some issue with the example pictures.
